pyproject.toml

```diff
@@ -2,13 +2,15 @@
 # Minimum requirements for the build system to execute.
 # See https://github.com/scipy/scipy/pull/12940 for the AIX issue.
 requires = [
-    "setuptools>=61.0.0",
+    "meson-python==0.13.1",
+    "meson[ninja]==1.0.1",
```

Comment:
`[ninja]` should be removed here; `meson-python` already adds that if it's missing (but prefers the system version if installed). 

Also, these pins are typically not what you want in `main`, only in a release branch. Or better, an upper bound and a comment that it's for safety and not because of known incompatibilities - that way packagers know that a looser bound or no bound is fine if they are otherwise running into conflicts.